### ISSUE TITLE
Wrong class export

### ISSUE THREAD

**Issue by mateus4k** [avoidwork/tiny-lru#202]
Hi, first of all, thanks for making this available as an npm module. When using this library I noticed that the exported class is actually replaced by the provided `.d.ts` file, causing `LRU` to be interpreted by TypeScript as an interface instead of a class.

```ts
import { LRU } from "tiny-lru";
const cache = new LRU(max, ttl = 0, resetTtl = false);
// 'LRU' only refers to a type, but is being used as a value here.ts(2693)
```

The only way to use the library in v11.2.6 is using the factory:
```ts
import { lru } from "tiny-lru";
```

I would like to suggest that the interface be changed to `LRUInstance`, and then the class can be exported with the name `LRU`. Exporting the class or factory as default (using `export default`) might also be a good idea.

**Comment by avoidwork** [comment]
Hi,

This should be resolved with 11.2.7, which had the `lru.d.ts` updated with the corrected export. Verified locally with this basic script

```javascript
import { LRU } from "tiny-lru";
const cache = new LRU(500, 0, false);
cache.set('abc', true);
console.log(cache.get('abc'));
```

The generic seems is inferred in my IDE

![Screenshot 2024-07-05 225332](https://github.com/avoidwork/tiny-lru/assets/369698/881b7eae-bec2-4950-956d-bb8fb674ef54)

**Comment by mateus4k** [comment]
Thanks!
